<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-C4GO84FG" failures="2" tests="16" name="Test" time="35.763" errors="0" timestamp="18 May 2019 19:30:04 GMT">
  <testcase classname="com.TestNG.pckg.ExceptionTimeOutTest" name="infiniteLoopTest" time="2.006">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.TestNG.pckg.ExceptionTimeOutTest.infiniteLoopTest() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.TestNG.pckg.ExceptionTimeOutTest.infiniteLoopTest() didn't finish within the time-out 2000
at java.base@11.0.2/java.io.FileOutputStream.writeBytes(Native Method)
at java.base@11.0.2/java.io.FileOutputStream.write(FileOutputStream.java:354)
at java.base@11.0.2/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
at java.base@11.0.2/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
at java.base@11.0.2/java.io.PrintStream.write(PrintStream.java:561)
at java.base@11.0.2/sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:233)
at java.base@11.0.2/sun.nio.cs.StreamEncoder.implFlushBuffer(StreamEncoder.java:312)
at java.base@11.0.2/sun.nio.cs.StreamEncoder.flushBuffer(StreamEncoder.java:104)
at java.base@11.0.2/java.io.OutputStreamWriter.flushBuffer(OutputStreamWriter.java:184)
at java.base@11.0.2/java.io.PrintStream.write(PrintStream.java:606)
at java.base@11.0.2/java.io.PrintStream.print(PrintStream.java:676)
at java.base@11.0.2/java.io.PrintStream.println(PrintStream.java:812)
at app//com.TestNG.pckg.ExceptionTimeOutTest.infiniteLoopTest(ExceptionTimeOutTest.java:13)
at java.base@11.0.2/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base@11.0.2/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@11.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base@11.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base@11.0.2/java.lang.Thread.run(Thread.java:834)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- infiniteLoopTest -->
  <testcase classname="com.TestNG.pckg.TestNGBasics2" name="googleLogoTest" time="0.114"/>
  <testcase classname="com.TestNG.pckg.TestNGBasics2" name="googleSearchTest" time="0.306"/>
  <testcase classname="com.TestNG.pckg.TestNGBasics2" name="googleTitleTest" time="0.142"/>
  <testcase classname="com.TestNG.pckg.GoogleTest" name="googleLogoTest" time="0.048"/>
  <testcase classname="com.TestNG.pckg.GoogleTest" name="googleTitleTest" time="0.094"/>
  <testcase classname="com.TestNG.pckg.GoogleTest" name="searchTest" time="0.109"/>
  <testcase classname="com.TestNG.pckg.InvocationCount" name="sum" time="0.092"/>
  <testcase classname="com.TestNG.pckg.InvocationCount" name="sum" time="0.0"/>
  <testcase classname="com.TestNG.pckg.InvocationCount" name="sum" time="0.0"/>
  <testcase classname="com.TestNG.pckg.InvocationCount" name="sum" time="0.021"/>
  <testcase classname="com.TestNG.pckg.InvocationCount" name="sum" time="0.089"/>
  <testcase classname="com.TestNG.pckg.TestNGFeatures" name="test1" time="0.079"/>
  <testcase classname="com.TestNG.pckg.TestNGFeatures" name="test2" time="0.121"/>
  <testcase classname="com.TestNG.pckg.TestNGBasics" name="test1" time="1.477"/>
  <testcase classname="com.TestNG.pckg.TestNGBasics" name="test2" time="3.481">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;true&amp;gt; but was:&amp;lt;Gmail&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<true> but was:<Gmail>
at com.TestNG.pckg.TestNGBasics.test2(TestNGBasics.java:74)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Test -->
